roles/confluent.kerberos/tasks/main.yml

```diff
@@ -1,15 +1,15 @@
 ---
-- name: Install the kerberos packages
-  yum:
-    state: present
-    name: "{{ kerberos.redhat_packages }}"
-  when: ansible_os_family == "RedHat"
-
-- name: Install the kerberos packages
-  apt:
-    state: present
-    name: "{{ kerberos.ubuntu_packages }}"
-  when: ansible_os_family == "Debian"
+# - name: Install the kerberos packages
+#   yum:
+#     state: present
+#     name: "{{ kerberos.redhat_packages }}"
+#   when: ansible_os_family == "RedHat"
```

Comment:
@JumaX services are starting up without these packages install on centos and ubuntu

Reply:
@domenicbove which versions of CentOS and Ubuntu?  Previous versions required these to be installed.

Reply:
confirmed ubuntu does not need the packages either, ill remove the commented out code